dashboard/app/views/levels/_teacher_panel.html.haml

```diff
@@ -13,7 +13,7 @@
   - data[:page_type] = 'level' if @level.present?
 
 - if @level && @script_level
-  - if @level.try(:examples).present? && current_user.authorized_teacher? # 'solutions' for applab-type levels
+  - if @level.try(:examples).present? && (@current_user.authorized_teacher? || @script.csf?)# 'solutions' for applab-type levels
```

Comment:
@dmcavoy, i added the `@` to `current_user` here for consistency with the rest of the file, but please let me know (and explain) if there was a reason you were referencing the global object.

Reply:
No reason. Probably didn't realize I was referencing the global.

Reply:
👍 thanks Dani!